tutorials/getting_started/so_you_built_a_model.md

```diff
@@ -0,0 +1,182 @@
+                                      [--output-file OUTPUT_FILE]
+                                      [--batch-size BATCH_SIZE]
+                                      [--silent]
+                                      [--cuda-device CUDA_DEVICE]
+                                      [-o OVERRIDES]
+                                      archive_file input_file
+```
+
+In the simplest case, just specify the `archive_file`
+(that's the trained model we downloaded above)
+and the `input_file`
+(that's the JSON lines file above)
+and your predictions will get printed to the console.
+If you'd rather have them in a file, specify the `--output-file` parameter.
+
+It's also possible to use non-JSON formats, for example, if you had CSV data.
+In this case your `Predictor` would need to override `load_line` and `dump_line`.
+
+## Running a Web Demo
+
+Once you have a trained model and a predictor,
+it's easy to run a simple web demo
+using `allennlp.service.server_simple.make_app()`:
+
+```python
+from allennlp.models.archival import load_archive
+from allennlp.service.predictors import Predictor
+from allennlp.service.server_simple import make_app
+
+from gevent.wsgi import WSGIServer
+
+archive = load_archive('/path/to/trained/bidaf/model.tar.gz')
+predictor = Predictor.from_archive(archive, 'machine-comprehension')
+
+app = make_app(predictor=predictor,
+               field_names=['passage', 'question'])
+
+http_server = WSGIServer(('0.0.0.0', 8888), app)
+http_server.serve_forever()
+```
+
+The important things to notice are
+
+* We had to explicitly specify which predictor to use. If you were
+  using a custom one, you'd need to import it at the top of the script.
+* We had to explicitly specify the field names. These are used to generate
+  the web interface but also (more importantly) as the keys in the input JSON,
+  which means they need to match what your `Predictor` expects
+* We specified port 8888, but you could have specified any other port.
+
+If you run this script and visit `localhost:8888` in your browser,
+you'll see a moderately attractive demo that displays the raw JSON
+results:
+
+![demo screenshot](simple_server.png)
+
+It's possible this won't quite suit your needs, in which case
+`make_app` [provides a few more options](https://github.com/allenai/allennlp/blob/master/allennlp/service/server_simple.py#L25)
+
+For example, you might want to change the page title to something other than "AllenNLP Demo".
+If, so, pass in a `title="Something else"` parameter.
+
+Or you might want to edit the returned JSON. Maybe you want to remove some of its verbose
+fields, or maybe you want to add a new nicely formatted field. In that case you can pass
+a `sanitizer=sanitizer` parameter, with a JSON-in-JSON-out function that contains the editing
+logic.
+
+If you're really motivated, you might want to just provide your own HTML page.
+(If so, I suggest using ours as a starting point, as it already contains the JavaScript
```

Comment:
I'd recommend removing the parentheses from this sentence and putting it at the end of the paragraph.  If you do this, you'd probably want to change "If so" to something that works better with the sentence being at the end.  Also, you used "we" at the top of the tutorial, and "I" here - it's probably best to be consistent in using one or the other.